.vscode/launch.json

```diff
@@ -89,6 +89,32 @@
             "request": "launch",
             "url": "http://localhost:8080/",
             "webRoot": "${workspaceRoot}"
+        },
+        {
+            // http://blog.likewise.org/2017/02/debugging-a-javascript-webdriverio-project-in-vscode/
+            // http://www.wiktorzychla.com/2017/08/debugging-javascript-webdriver-io-in.html
+            "type": "node",
+            "request": "launch",
+            "port": 5859,
+            "protocol": "inspector",
+            "name": "WebdriverIO",
+            "runtimeExecutable": "${workspaceRoot}/node_modules/.bin/wdio",
+            "windows": {
+                "runtimeExecutable": "${workspaceRoot}/node_modules/.bin/wdio.cmd"
+            },
+            "cwd": "${workspaceRoot}/examples/browser",
+            "console": "integratedTerminal",
```

Comment:
it should be replaced with:
https://github.com/theia-ide/theia/blob/ec299e503a8ee359672a6d2a48a694da7488e29d/.vscode/launch.json#L35-L37

also sourceMaps are missing

Reply:
I'm not sure how it works exactly, but omitting sourceMaps still allows me to set breakpoints, also I thought it was true by default?